scripts/tools-protobuf.sh

```diff
@@ -0,0 +1,7 @@
+curl -LO https://github.com/protocolbuffers/protobuf/releases/download/v3.12.0/protoc-3.12.0-linux-x86_64.zip
+unzip protoc-3.12.0-linux-x86_64.zip -d $HOME/.local
+export PATH="$PATH:$HOME/.local/bin"
+
+go get google.golang.org/grpc
+
+sudo apt-get install -y golang-protobuf-extensions-dev
```

Comment:
What is this for?

Reply:
You asked me to add golang protobuf extension, I thought this is the one we needed.

Reply:
github.com/protocolbuffers/protobuf-go

Reply:
Not that one either, please try this:
```

Reply:
Also please try to run ```make protoc``` after this to see if it works or not.